stdlib/http/src/main/java/org/ballerinalang/net/http/websocket/observability/WebSocketObservabilityConstants.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.ballerinalang.net.http.websocket.observability;
+
+/**
+ * Providing observability functionality to WebSockets.
+ *
+ * @since 1.1.0
+ */
+public class WebSocketObservabilityConstants {
+
+    public static final String TAG_CONNECTION_ID = "connectionID";
+    public static final String TAG_CLIENT_OR_SERVER = "client_or_server";
+    public static final String TAG_SERVICE = "service";
+    public static final String TAG_MESSAGE_TYPE = "type";
+    public static final String TAG_ERROR_TYPE = "error_type";
+    public static final String TAG_RESOURCE = "resource";
+
+    static final String[] METRIC_REQUESTS = { "requests", "Number of WebSocket connection requests"};
+    static final String[] METRIC_CONNECTIONS = {"connections", "Number of currenctly active connections"};
+    static final String[] METRIC_MESSAGES_RECEIVED = {"messages_received", "Number of messages received"};
+    static final String[] METRIC_MESSAGES_SENT = {"messages_sent", "Number of messages sent"};
+    static final String[] METRIC_ERRORS = {"errors", "Number of errors"};
+    static final String[] METRIC_RESOURCES_INVOKED = {"resources_invoked", "Number of resources invoked"};
+
+    static final String CLIENT_OR_SERVER_CLIENT = "client";
+    public static final String CLIENT_OR_SERVER_SERVER = "server";
```

Comment:
`CLIENT_OR_SERVER_SERVER` and `CLIENT_OR_SERVER_CLIENT` Can you explain the usage of these? Can't we just have `CLIENT` and `SERVER`?

Reply:
logic behind that was to indicate that these two were the possible values for the CLIENT_OR_SERVER tag. Fixed now to make that clearer.

New tag name is CONTEXT
Possible values are CONTEXT_SERVER AND CONTEXT_CLIENT